study-builder/studies/testboston/subs.conf

```diff
@@ -12,10 +12,10 @@
     "study": "testboston",
     "act": {
       "consent": "CONSENT",
-      "covid_survey": "COVID_SURVEY",
+      "baseline_covid": "BASELINE_COVID",
       "address": "ADDRESS",
-      "followup": "FOLLOWUP_SURVEY",
-      "symptoms_survey": "SYMPTOMS_SURVEY"
+      "baseline_symptom": "BASELINE_SYMPTOM",
+      "longitudinal": "LONGITUDINAL_COVID",
```

Comment:
Renaming because of https://broadinstitute.slack.com/archives/G012Y2S45KN/p1597701514218700?thread_ts=1597684886.169500&cid=G012Y2S45KN